internal/multigitter/shared.go

```diff
@@ -70,3 +72,57 @@ func ParseConflictStrategy(str string) (ConflictStrategy, error) {
 		return ConflictStrategyReplace, nil
 	}
 }
+
+// CreateTempDir creates a temporary directory in the given directory.
+// If the given directory is an empty string, it will use the os.TempDir()
+func CreateTempDir(cloneDir string) (string, error) {
```

Comment:
Does not have to be public since it's only used within this package

```suggestion
func createTempDir(cloneDir string) (string, error) {
```